### ISSUE TITLE
ClassDeclaration.getAllMembers does not return abstract methods

### ISSUE THREAD

**Issue by mnmtanish** [dsherret/ts-simple-ast#102]
When I load [this file](https://github.com/Cinergix/mockgen/blob/027cbcd9f07b9d3ea85979de3572678377420388/example/src/abstract.ts), get the class declaration and and call getAllMembers, it does not return abstract methods.

**Comment by dsherret** [comment]
Thanks for reporting this! It's because the [isImplementation()](https://github.com/dsherret/ts-simple-ast/blob/94a7568ece02150b8c6c557d9285d9b6353a4dae/src/compiler/function/OverloadableNode.ts#L57) method doesn't take into account abstract methods. I'll fix it soon.

A workaround in the meantime is to access the `members` property on the underlying compiler node:

```typescript
const members = classDeclaration.getNodeProperty("members");
```

**Comment by mnmtanish** [comment]
Thanks for mentioning the overloads. I was [using](https://github.com/Cinergix/mockgen/blob/master/src/index.ts#L355) the workaround but didn't think of overloads.

**Comment by dsherret** [comment]
Fixed in 0.78.1.

Fixed some filtering done in ClassDeclaration to handle this scenario. I decided to leave `.isImplementation()` as-is because abstract methods aren't really the implementation... so abstract methods will continue returning false for that. I don't think I was thinking properly this morning.

**Comment by mnmtanish** [comment]
Thanks a lot
